src/common/utils/model.ts

```diff
@@ -10,13 +10,15 @@ import { findCountyByFips } from 'common/locations';
 import moment from 'moment';
 import { RegionSummaryWithTimeseries } from 'api/schema/RegionSummaryWithTimeseries';
 import { assert } from '.';
+import { getSnapshotUrlOverride } from './snapshots';
 
 const cachedProjections: { [key: string]: Promise<Projections> } = {};
 export function fetchProjections(
   stateId: string,
   countyInfo: any = null,
   snapshotUrl: string | null = null,
 ) {
+  snapshotUrl = snapshotUrl || getSnapshotUrlOverride();
```

Comment:
I think it would be cleaner to have the caller of `fetchProjections` calling `getSnapshotUrlOverride` and passing the `snapshotUrl` parameter as usual instead? If this is much easier I think we can leave it though

Reply:
I'm kinda' on the fence on this.  Right now most of our code isn't snapshot-aware.  In our location page, etc. we just call `useProjections(state, [county])` and don't think about snapshots.  The only code that is currently snapshot-aware is the compare tool, since it's comparing snapshots.

I think I'd like to keep it that way, meaning that this snapshot override just happens transparently without the calling code knowing about it.  

But I can see both points-of-view.  It's a little "magical" as-is which means somebody could accidentally break it or whatever.

Reply:
makes sense 👍